site/components/src/main/java/uk/nhs/digital/intranet/utils/AccessTokenEncoder.java

```diff
@@ -0,0 +1,40 @@
+package uk.nhs.digital.intranet.utils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.nhs.digital.intranet.model.AccessToken;
+
+import java.io.*;
+import java.util.Base64;
+
+public class AccessTokenEncoder {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(AccessTokenEncoder.class);
+
+    public String encode(final AccessToken accessToken) {
```

Comment:
It's only a style suggestion, but I would have made this method static.

Reply:
It could be static, however, I wanted to keep using Spring beans, since all the other classes with regards to the people search are Spring beans (and they're easier to mock in unit tests)